### ISSUE TITLE
Does the URL Scheme of IINA support custom titles

### ISSUE THREAD

**Issue by wisinfun** [iina/iina#4508]
Does the URL Scheme of IINA support custom titles？
like this `iina://weblink?url={{url}}&title={{title}}`

I checked the wiki and issues and found no relevant clues

**Comment by low-batt** [comment]
No. IINA does not provide a way to customize the window's title at this time.

I will label this as a feature request.

**Comment by njzydark** [comment]
use `mpv_force-media-title` can set custom title
